package/rules/node_modules.js

```diff
@@ -6,7 +6,7 @@ const { nodeEnv } = require('../env')
 module.exports = {
   test: /\.(js|mjs)$/,
   include: /node_modules/,
-  exclude: /@babel(?:\/|\\{1,2})runtime/,
+  exclude: /@babel(?:\/|\\{1,2})runtime|core-js/,
```

Comment:
running babel on core-js is causing [issues](https://github.com/zloirock/core-js/issues/514)